src/main/java/com/redhat/devtools/lsp4ij/features/documentSymbol/DocumentSymbolData.java

```diff
@@ -53,6 +54,7 @@ public DocumentSymbolData(@NotNull DocumentSymbol documentSymbol,
         this.psiFile = psiFile;
         this.languageServer = languageServer;
         this.parent = parent;
+        this.textRange = LSPIJUtils.toTextRange(documentSymbol.getRange(), LSPIJUtils.getDocument(psiFile.getVirtualFile()));
```

Comment:
Don't compute this textRange is the constructor since this text range could ne not used, but compute it if the method getTextRange is called.

```java
Override
public TextRange getTextRange() {
    if (textRange == null) {
        textRange = LSPIJUtils.toTextRange(documentSymbol.getRange(), LSPIJUtils.getDocument(psiFile.getVirtualFile()));
   }
    return textRange;
}
```